doc/src/documentation/architecture/security/README.md

```diff
@@ -2,29 +2,35 @@
 
 For the security concept, we identified the [personas](#personas) listed below.
 Then we determined which persona is allowed to use which feature.
-The results of this process are listed under [features](#features)
+The results of this process are listed under [features](#features-and-the-personas-that-can-use-it)
 
 ## Personas
 
 - [örtliche Ausbildungsleitung](../../../glossary.md#ortliche-ausbildungsleitung)
 - [örtliche Ausbilderin](../../../glossary.md#ortliche-ausbilderin)
 - [Nwk](../../../glossary.md#nwk)
 
-## Features
+## Features and the Personas that can use it
 
 - [Nachwuchskräfte](../../../features/Nachwuchskraefte.md) (örtliche Ausbildungsleitung)
 - [Meldezeitraum](../../../features/meldezeitraum.md) (örtliche Ausbildungsleitung)
-- [Meldung örtl. Ausbilder](../../../features/Praktikumsplaetze.md) (örtliche Ausbilderin / anyone except Nwk)
-- [Meldung örtl. Ausbildungsleitung](../../../features/Praktikumsplaetze.md) (örtliche Ausbildungsleitung)
-- [Zuweisung](../../../features/Zuweisung.md) (örtliche Ausbildungsleitung)
+- [Submission örtl. Ausbilder](../../../features/Praktikumsplaetze.md#submission-örtliche-ausbilder) (örtliche Ausbilderin)
+- [Submission örtl. Ausbildungsleitung](../../../features/Praktikumsplaetze.md#submission-örtliche-ausbildungsleitung) (örtliche Ausbildungsleitung)
+- [Assignment](../../../features/Zuweisung.md) (örtliche Ausbildungsleitung)
 
 ## Technical Implementation
```

Comment:
please describe the implementation in the backend too. We should make clear that security is done by backend. Frontend-Security-Features only usability features.